### PR TITLE
Adding Shared Responsibility Model Overview

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

Adding Chaingaurd Shared Responsibility Model doc

### What should this PR do?

resolves https://github.com/chainguard-dev/internal/issues/3911

### Why are we making this change?

@mattmoor drafted a newer, shorter version of the initial CSRM doc. This PR brings it closer to Academy's style and cleans up some of the images.

original is here: https://docs.google.com/document/d/1I7SdlMJFJFW7ufoCfP5H_oOY6GzxWVTkalz5NnEmFfc/edit?usp=sharing

### What are the acceptance criteria? 

Should make sense as written, images should be legible. Note that the content had been reviewed and approved for technical accuracy  and I changed very little in terms of content. Most of my changes from the original were just to make it more readable on EDU.

### How should this PR be tested?

No tech test, just a close read would be great. Thanks!

### CODE REVIEW

File: content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<br /> 
+
+Upstream projects are responsible for staying on API-compatible versions of libraries. Chainguard is responsible for rebuilding the upstream project [with the latest toolchain](https://www.chainguard.dev/unchained/chainguard-patches-3-silent-golang-cves-in-under-24-hours), and patching static and dynamic dependencies where such a change is non-breaking. Customers are responsible for tracking a supported version of the Chainguard image.
+
+
+### Base Images
+
+These are Chainguard images that users extend with their own packages and applications (such as with a Dockerfile). We created the following diagram to clarify where the division of responsibility is drawn for these images:
+
+<center><img src="csrm-3.png" alt="Diagram representing Chainguard's shared responsibility model for Base Images. The diagram takes the shape of a pyramid with 3 tiers: User App, Toolchains (go, java, php, …), and System (glibc, openssl, …). To the left of this pyramid are two brackets, showing that user apps (the custom code level) are serviced by customers and the Toolchains and System levels are serviced by Chainguard." style="width:821px;"></center>
```
Comment (mattmoor) [inline content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md:69]: This one's wrong

Comment (mattmoor) [reply content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md:69]:
```suggestion
<center><img src="csrm-4.png" alt="Diagram representing Chainguard's shared responsibility model for Base Images. The diagram takes the shape of a pyramid with 3 tiers: User App, Toolchains (go, java, php, …), and System (glibc, openssl, …). To the left of this pyramid are two brackets, showing that user apps (the custom code level) are serviced by customers and the Toolchains and System levels are serviced by Chainguard." style="width:821px;"></center>
```

Comment (SharpRake) [reply content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md]: thanks for catching this! lots of back and forth with adding white backgrounds then getting these images centered

**Review by erikaheidi — APPROVED** [review]
Everything looking good to me. Very well explained and clear, although it can read a bit like a contract (which is fine) so the diagrams really help to make it easier to digest. Maybe in the future we can find a way to summarize this in one big table or diagram as a TLDR at the top, but it's just an idea for a future improvement. I believe this is good to be merged.

File: content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+<br /> 
+
+Upstream projects are responsible for patching supported releases in a timely manner. Chainguard is responsible for releasing fully patched toolchain and base images. Customers are responsible for patching any applications and dependencies they add to a Chainguard image. We recommend using a fully patched Chainguard toolchain image to build the application, and using a fully patched Chainguard base image to layer the final application on.
+
+
+## Testing
+
+Upstream projects are responsible for defining conformance criteria and establishing conformance benchmarks (such as [Java TCK](https://en.wikipedia.org/wiki/Technology_Compatibility_Kit), [Kubernetes Conformance](https://github.com/cncf/k8s-conformance/blob/master/instructions.md), [Knative Conformance](https://github.com/knative/specs)) that clearly outline the criteria for a downstream distribution to represent itself as a "conformant" distribution of the software.
```
Comment (mattmoor) [inline content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md:77]: I think we should change the TCK reference to JCK (for consistency) and link to our blog post here instead: https://www.chainguard.dev/unchained/chainguards-openjdk-java-images-are-now-jck-conformant

File: content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+### Base Images
+
+These are Chainguard images that users extend with their own packages and applications (such as with a Dockerfile). We created the following diagram to clarify where the division of responsibility is drawn for these images:
+
+<center><img src="csrm-4.png" alt="Diagram representing Chainguard's shared responsibility model for Base Images. The diagram takes the shape of a pyramid with 3 tiers: User App, Toolchains (go, java, php, …), and System (glibc, openssl, …). To the left of this pyramid are two brackets, showing that user apps (the custom code level) are serviced by customers and the Toolchains and System levels are serviced by Chainguard." style="width:821px;"></center>
```
Comment (mattmoor) [inline content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md:69]: Does this not auto-deploy?  I still see the old image 🤔

**Review by mattmoor — APPROVED** [review]

File: content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md (hunk 3 above)
Comment (SharpRake) [reply content/chainguard/chainguard-images/recommended-practices/shared-responsibility-model/index.md:69]: It looks right for me. Could be a caching issue?

Regardless, the deploy preview is not 100% perfect

**Pull request merged**